greet.js

```diff
@@ -0,0 +1,11 @@
+function greet(name) {
+  if(typeof(name) === 'string') {
+    return 'Hello ' + name;
+  } else {
+    return 'Please enter a string';
```

Comment:
We haven't talked about errors yet, but this would be a great place to throw an error, which you can then test for in your tests. :-)